pages/login.tsx

```diff
@@ -149,7 +163,18 @@ export default function Login() {
                   Muista minut
                 </Label>
               </div>
-              <Button type="submit">Kirjaudu sisään</Button>
+              {loginQuery.isFetching ? (
+                <Button
+                  type="submit"
+                  className="cursor-not-allowed bg-red-500"
+                  disabled
+                >
+                  Kirjaudutaan sisään
+                  <span className="loading-dots absolute" />
+                </Button>
+              ) : (
+                <Button type="submit">Kirjaudu sisään</Button>
+              )}
```

Comment:
Tein tämmöisen pienen if/else-renderöinnin. 

Jos kirjautumisyritys on tehty, estetään napin painaminen, vaihdetaan väri (ei välttämättä hyvä idea, mutta olkoon tässä alussa nyt näin) ja laitetaan kursoriksi `not-allowed`. Vaihdoin tekstin `Kirjaudutaan sisään` ja sitten `span`illa nuo pistelatausindikaattori tai millä nimellä haluaa kutsuakkaan :smile: 

Syy `absolute`-classNamen käyttöön on se, että ilman sitä `Kirjaudutaan sisään` -teksti liikkui edestakaisin pisteiden mukaan. Se ei näyttänyt hyvältä :sweat_smile: 

Jos ei ole kirjauduttu niin sitten ihan tavallinen kirjautumisnappula näytetään :+1: 


Onko ihan ok noin vai pitäisikö tehdä oma sisäinen komponentti tmv? :smile: